<ui version="4.0" >
 <class>BoundaryConditionsVelocityInletForm</class>
 <widget class="QWidget" name="BoundaryConditionsVelocityInletForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBoxVelocity" >
     <property name="title" >
      <string>Velocity</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" name="_2" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_3" >
          <item>
           <widget class="QComboBox" name="comboBoxVelocity" />
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>16</width>
              <height>22</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditVelocity" />
          </item>
          <item>
           <widget class="QLabel" name="labelUnitVelocity" >
            <property name="text" >
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>22</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frameVelocity" >
        <property name="minimumSize" >
         <size>
          <width>16</width>
          <height>42</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth" >
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="_4" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>21</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="_5" >
           <item>
            <widget class="QLabel" name="labelXVelocity" >
             <property name="text" >
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditXVelocity" >
             <property name="maximumSize" >
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelYVelocity" >
             <property name="text" >
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditYVelocity" >
             <property name="maximumSize" >
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelZVelocity" >
             <property name="text" >
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditZVelocity" >
             <property name="maximumSize" >
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
